csharp/src/Ice/StringUtil.cs

```diff
@@ -602,13 +602,60 @@ public static string UnescapeString(string s, int start, int end, string special
             return l.ToArray();
         }
 
+        public static string JoinStringProperty(IReadOnlyList<string> values)
+        {
+            string delimiters = ", \n\r\t";
```

Comment:
Could we make this an array now? I think it would make space more obvious " ". All we do is iterate over it anyway.

Reply:
Yes fixed